packages/google_sign_in/google_sign_in/lib/google_sign_in.dart

```diff
@@ -251,6 +244,7 @@ class GoogleSignIn {
     final Completer<void> completer = Completer<void>();
     future.whenComplete(completer.complete).catchError((dynamic _) {
       // Ignore if previous call completed with an error.
+      // TODO: Should we log errors here, if debug or similar?
```

Comment:
Yes. let's log an error here. Personally, I think we should be clear about things like this error is completely harmless and it is logged for debug purpose only.
@mehmetf I saw that this is added in https://github.com/flutter/plugins/commit/4eeb744bfce3cbacd64dcce32b52295c3299fe89
Did you recall why did we ignore this error?

Reply:
I'll revisit this to add logging, so you guys can discuss how much of it we need :)

(I'd recommend that this gets logged at the lowest level that makes sense, like trace or similar)